plugin.yaml

```diff
@@ -421,7 +421,7 @@ node_types:
 
   # distributed logical routers
   cloudify.nsx.dlr:
-    derived_from: cloudify.nodes.Router
+    derived_from: cloudify.nodes.Compute
```

Comment:
This seems like it should derive from Router rather than Compute (and thus not need the install_agent stuff below)- what's the reason for deriving it from Compute instead?